### PR TITLE
[Merged by Bors] - Add docs on how to connect to Trino

### PR DESCRIPTION
**Pull request by sbernauer**
# Description

Closes https://github.com/stackabletech/trino-operator/issues/321

## Definition of Done Checklist

- Not all of these items are applicable to all PRs, the author should update this template to only leave the boxes in that are relevant
- Please make sure all these things are done and tick the boxes
 
```[tasklist]
# Author
- [ ] Changes are OpenShift compatible
- [ ] CRD changes approved
- [ ] Helm chart can be installed and deployed operator works
- [ ] Integration tests passed (for non trivial changes)
```

```[tasklist]
# Reviewer
- [ ] Code contains useful comments
- [ ] (Integration-)Test cases added
- [ ] Documentation added or updated
- [ ] Changelog updated
- [ ] Cargo.toml only contains references to git tags (not specific commits or branches)
```

```[tasklist]
# Acceptance
- [ ] Feature Tracker has been updated
- [ ] Proper release label has been added
```

Once the review is done, comment `bors r+` (or `bors merge`) to merge. [Further information](https://bors.tech/documentation/getting-started/#reviewing-pull-requests)

### CODE REVIEW

File: docs/modules/trino/pages/usage_guide/connect_to_trino.adoc [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+A https://trino.io/docs/current/client/jdbc.html[Open source JDBC driver] does exist as well as a proprietary https://docs.starburst.io/data-consumer/clients/odbc.html[ODBC driver].
+
+Multiple SQL clients can be used to connect to Trino as described below.
+
+== Perquisites
+The Trino server must be reachable from your local computer.
+It is normally secured via HTTPS with a certificated provided by a xref:secret-operator:secretclass.adoc[SecretClass].
+
+As most setups use a self-signed certificate, the easiest way is to disable the Trino cert validation as the following guide will do.
+A production-ready solution would be to extract the `ca.crt` from the SecretClass and provide it to the Trino client.
```
Comment (fhennig) [inline docs/modules/trino/pages/usage_guide/connect_to_trino.adoc:13]: > A production-ready solution would be to ...

We should explain that part eventually too.

File: docs/modules/trino/pages/usage_guide/connect_to_trino.adoc [hunk 2]
```diff
@@ -0,0 +1,75 @@
+= Connect to Trino
+
+Trino offers a SQL endpoint via a HTTP Api.
+A https://trino.io/docs/current/client/jdbc.html[Open source JDBC driver] does exist as well as a proprietary https://docs.starburst.io/data-consumer/clients/odbc.html[ODBC driver].
+
+Multiple SQL clients can be used to connect to Trino as described below.
+
+== Perquisites
+The Trino server must be reachable from your local computer.
+It is normally secured via HTTPS with a certificated provided by a xref:secret-operator:secretclass.adoc[SecretClass].
```
Comment (fhennig) [inline docs/modules/trino/pages/usage_guide/connect_to_trino.adoc:10]:
```suggestion
It is typically secured via HTTPS with a certificated provided by a xref:secret-operator:secretclass.adoc[SecretClass].
```

File: docs/modules/trino/pages/usage_guide/connect_to_trino.adoc [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+Please have a look at the https://trino.io/docs/current/client/cli.html[official trino-cli docs].
+Please don't forget to specify the `--insecure` flag, so you command looks something like:
+
+[source,console]
+----
+java -jar ~/Downloads/trino-cli-403-executable.jar --server https://85.215.195.29:8443 --user admin --password --insecure
+----
+
+== Connect with DBeaver
+https://dbeaver.io[DBeaver] is a free multi-platform database tool for all people who need to work with databases.
```
Comment (fhennig) [inline docs/modules/trino/pages/usage_guide/connect_to_trino.adoc:25]:
```suggestion
https://dbeaver.io[DBeaver] is a free multi-platform database tool for anyone who needs to work with databases.
```

File: docs/modules/trino/pages/usage_guide/connect_to_trino.adoc [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+It is installed locally and can connect to Trino and offers a convenient UI to explore and work with Trino.
+
+First of you need to click on the `New Database Connection` icon, select `Other` -> `Trino`
+
+image::connect-with-dbeaver-1.png[]
+
+Afterwards you need to enter the `Host` as well as the `Port` as well as `Username` and `Password`.
```
Comment (fhennig) [inline docs/modules/trino/pages/usage_guide/connect_to_trino.adoc:32]:
```suggestion
Afterwards you need to enter the `Host` and `Port` as well as `Username` and `Password`.
```

File: docs/modules/trino/pages/usage_guide/connect_to_trino.adoc [hunk 5]
```diff
@@ -0,0 +1,75 @@
...
+
+image::connect-with-dbeaver-2.png[]
+
+After entering the details you need to switch to the tab called `Driver properties`.
+First action is to click on the `SSL` driver property and enter the value `true`.
+Additionally you need to click on the green plus icon on the bottom left corner to add a User Property called `SSLVerification`.
```
Comment (fhennig) [inline docs/modules/trino/pages/usage_guide/connect_to_trino.adoc:38]:
```suggestion
Additionally you need to click on the green plus icon in the bottom left corner to add a User Property called `SSLVerification`.
```

**Review by fhennig — COMMENTED** [review]
I think that's it for now.

Could you also move this doc in the navigation as the second item? Right after "Creating a trino cluster", I think that's the best spot for it

File: docs/modules/trino/pages/usage_guide/connect_to_trino.adoc (hunk 3 above)
Comment (sbernauer) [reply docs/modules/trino/pages/usage_guide/connect_to_trino.adoc:25]: Copied that quote from their website but I'm fine with the suggestion ^^

**Comment by fhennig** [comment]
bors merge

**Pull request closed without merging**